specification/operationalinsights/resource-manager/readme.md

```diff
@@ -38,6 +38,35 @@ openapi-type: arm
 tag: package-2023-09
 ```
 
+### Tag: package-2023-09
+
+These settings apply only when `--tag=package-2023-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-01-preview'
```

Comment:
>package-2023-01-previe [](http://example.com/codeflow?start=19&length=22)

I see there is a stable version 2023-09-01, shouldn't the preview version be 2024-01-01?